docs/versioning.rst

```diff
@@ -0,0 +1,55 @@
+*****************
+Versioning Policy
+*****************
+
+The internal interface of the `ddtrace` library is defined as follows:
+    Any module, function, class or attribute is internal if it is prefixed with a single underscore
+
+    Any package that is named `internal` is internal
```

Comment:
```suggestion
    Any package that is named ``internal`` is internal
```

Reply:
Should we just rename `internal` to `_internal`?

We should also call out and underscore prefix `_vendor` as well?

Reply:
Yep, using the single underscore prefix as the only criteria for what constitutes an internal interface would simplify this section.